_posts/2020-01-29-Introduction_to_the_IAC_Team.md

```diff
@@ -0,0 +1,121 @@
+Two members of the IAC team will take it in turns to review community contributors to Puppet Forge modules.
+We're there to ensure that the change meets the quality standard required but more important, we're there to help contributors understand how to do that.
+We're very fortunate to receive lots of high quality contributions or those willing to act upon feedback.
+The goal is to ensure that the quality of the Puppet modules remains high, whilst constantly being enhanced and reactive to the needs of the community.
+ 
+ 
+### Tech Stack
+Some folks may be curious as to what we use day-to-day to get the job done.
+There is quite a large amount of software, platforms and tools we use, some you'll likely have heard of and some internal proprietary tools.
+For internal tooling, we're in a constant effort to evolve them and make them available for use by the Puppet Community.
+
+### Development and Debugging
+For the most part, the IDE of choice for us is VSCode.
+It has support for Puppet manifests via [this extension](https://marketplace.visualstudio.com/items?itemName=jpogran.puppet-vscode).
+Some of the team also use RubyMine and Sublime Text can also be found on a lot of laptops.
+The [Ruby gem pry](https://rubygems.org/gems/pry/) is a heavily utilised one.
+This has proven an invaluable tool for debugging during development or investigation when reproducing issues reported to us.
+ 
+ 
+### Testing and CI Tools
+#### Litmus
+Litmus! Litmus is a fantastic resource and probably the most valuable productivity tools that we use day-to-day.
+I encourage you to check out the [Wiki for Litmus](https://github.com/puppetlabs/puppet_litmus/wiki) if you're thinking of becoming a module contributor.
+Litmus is a tool that allows us to quickly set up, execute and tear down a test environment for our modules.
+You can provision your test environment using Docker or Vagrant.
+For the IAC team, we have access to vmpooler, a proprietary tool/API that spins up and tears down virtual machines in our vSphere infrastructure.
+Within as little as seconds, I can have an environment ready to run acceptance tests on all compatible OSs for a given module, or I can have an environment to replicate a specific customer environment (e.g. on Windows Server 2012 R2, with the Scheduled Task module, I was seeing an issue with...).
+ 
+ 
+#### RSpec and Puppet RSpec
+RSpec is the test framework we use to write and execute our unit and acceptance tests.
+However, we also have the [RSpec-Puppet gem](https://rspec-puppet.com/) which provides a lot of helper methods to make life easier when testing modules.
+ 
+ 
+#### CI Tools
+We use a number of CI automation servers: Jenkins, Travis, AppVeyor and most recently Github Actions.
+There are different requirements for the variety of modules we have either authored, support or both, but we are constantly evaluating the best tools / platform for our use cases and those of the Puppet Community.
```

Comment:
```suggestion
There are different requirements for the variety of modules we have either authored, support. or both, and we are constantly evaluating the best tools / platform for our use cases and those of the Puppet Community.
```

Reply:
Github not picking up the change, but has been addressed.